Source/Cogworks.Umbraco.Essentials/Controllers/Actions/ResponsiveImageSurfaceController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Web.Mvc;
+using Cogworks.Essentials.Constants;
+using Cogworks.Essentials.Extensions;
+using Cogworks.Umbraco.Essentials.Builders.Interfaces;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Web.Mvc;
+
+namespace Cogworks.Umbraco.Essentials.Controllers.Actions
+{
+    public class ResponsiveImageSurfaceController : SurfaceController
+    {
+        private readonly IResponsiveImageBuilder _responsiveImageBuilder;
+
+        public ResponsiveImageSurfaceController(IResponsiveImageBuilder responsiveImageBuilder) => _responsiveImageBuilder = responsiveImageBuilder;
```

Comment:
Move `=> _responsiveImageBuilder = responsiveImageBuilder;` to new line, its to long